packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -377,14 +294,19 @@ export class ColorPicker extends LitElement implements InteractiveComponent, Loa
     this.listen("keyup", this.handleChannelKeyUpOrDown, { capture: true });
   }
 
+  connectedCallback(): void {
+    if (this.hasUpdated) {
```

Comment:
Could this check be moved inside `resizeCanvas`?

Reply:
I think so. I might be able to remove the `observeResize` in `firstUpdated` this way too.

Reply:
Yeah that might be better 👍